image_services/egis/make_egis_services.py

```diff
@@ -192,16 +197,17 @@ def main():
 
     arcpy.env.parallelProcessingFactor = '75%'
 
-    rasters = get_rasters(bucket, config['s3_prefix'], config['s3_suffix'])
-    update_csv(csv_file, rasters)
+    try:
```

Comment:
I think we can move this `try` back to it's original location, right before creating the geodatabase?